sql/moz-fx-data-marketing-prod/google_search_console_derived/site_impression_exports_empty_check_v1/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+friendly_name: Site Impression Exports Empty Check
+description: |-
+  Raises an error if any of the Google Search Console site impression exports are missing data for the given date.
+owners:
+- srose@mozilla.com
+scheduling:
+  dag_name: bqetl_google_search_console
+  destination_table: null
+  query_file_path: sql/moz-fx-data-marketing-prod/google_search_console_derived/site_impression_exports_empty_check_v1/query.sql
+  # Google Search Console exports typically happen two days after the data date in UTC.
+  parameters:
```

Comment:
For consistency should we do it this way?
https://github.com/mozilla/bigquery-etl/pull/4892/files#diff-ee64d2d3510eed266e548988757bc4c5b0b39871cfe551fa9118d9014ad2b4dbR23

Reply:
I did test using `date_partition_offset` in this case, and it didn't work properly with `destination_table: null`.  While the `bqetl` code could theoretically be adjusted to support that combination, IMO it doesn't really make sense to do so.  `date_partition_offset` makes sense when there's an actual date partition being written to, but there isn't a destination table or partition in this case.